### DISCUSSION TITLE
Can we setup git proxy for Azure DevOps Repos

### DISCUSSION THREAD

**Discussion by amalpp** [finos/git-proxy#602] [General]
can we setup this proxy server to Azure DevOps repos. if it is possible please help me to configure. 

Thanks in Advance

**Comment by JamieSlome** [comment]
@amalpp - absolutely. Anything that is git under the hood is supported by GitProxy 👍 

Have you tried following our [quickstart 🚀 guide](https://git-proxy.finos.org/docs/quickstart/intercept)?
